src/CCEdit/CCEdit.cpp

```diff
@@ -1245,6 +1293,15 @@ void CCEditMain::populateTilesets()
         for (const QString& file : path.entryList(tilesetGlob, QDir::Files | QDir::Readable, QDir::Name))
             tilesets << path.absoluteFilePath(file);
     }
+
+    // Search starting from the build directory when using CLion
+    path.setPath(QApplication::applicationDirPath());
+    for(int i = 0; i < 3; ++i) {
+        path.cdUp();
+    }
+    path.cd(QStringLiteral("share/cctools"));
+    for (const QString& file : path.entryList(tilesetGlob, QDir::Files | QDir::Readable, QDir::Name))
+        tilesets << path.absoluteFilePath(file);
```

Comment:
This doesn't really belong here...  Personally, I just copy the necessary files into the build dir to make it happy for debugging.